mars/serialize/dataserializer.py

```diff
@@ -389,6 +389,11 @@ def _deserialize_arrow_string_array(obj):
     return ArrowStringArray(pyarrow.chunked_array(obj))
 
 
+def _deserialize_arrow_list_array(obj):
+    from ..dataframe.arrays import ArrowListArray
```

Comment:
Is it possible that we store ``dtype.str`` with data chunks together and get type class with sth like ``ArrowArray.type('arrow[string]')(pyarrow.chunked_array(obj))``?

Reply:
OK, I will serialize the name as well  as data